roles/pre-flight-checks/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- debug: msg="running pre_flight_checks():"
+
+###########################################################################################
+## validate free space in filesystem that the Cinder temp directory lives on
+###########################################################################################
```

Comment:
it would also be good to have a glance disk size check to ensure we at least have enough space to download the images we give them.